CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Remove styles not in the Spec.
```

Comment:
Que penses-tu d'avoir un commentaire plus précis sur le composant impacté ? `Remote :hover styles to ListTable.Row.` un truc du style.

Reply:
ah mais complètement !